README.md

```
@@ -18,15 +24,36 @@ React MaterialUI nx ekans GraphQL git metapod NextJS discord.js vite nodeJS emot
+
+Last updated Feb 2025
 
-### Main site https://frzyc.github.io/genshin-optimizer/
+### Genshin Optimizer site https://frzyc.github.io/genshin-optimizer/
 
 - New characters and content will continue to be added as Genshin updates
 - ~~Artifact upgrade damage probability calculator - Q1 2024~~
 - ~~Teams + Loadouts - Q2 2024~~
 - ~~Optimal substat solver for TC mode - Q2 2024~~
 
+### Zenless Optimizer(alpha) site https://frzyc.github.io/zenless-optimizer/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)